core/src/main/scala/org/apache/spark/h2o/H2OContextImplicits.scala

```diff
@@ -29,7 +31,7 @@ abstract class H2OContextImplicits {
 
   protected def _h2oContext: H2OContext
   /** Implicit conversion from RDD[Supported type] to H2OFrame */
-  implicit def asH2OFrameFromRDDProduct[A <: Product : TypeTag](rdd : RDD[A]): H2OFrame = _h2oContext.asH2OFrame(rdd, None)
+  implicit def asH2OFrameFromRDDProduct[A <: Product : ClassTag : TypeTag](rdd : RDD[A]): H2OFrame = _h2oContext.asH2OFrame(rdd, None)
```

Comment:
We need `ClassTags` and `TypeTags` in order to create `H2OAwareRDD`. It failed on compile error without this change since the class tag wasn't available for `asH2OFrame` method